Makefile

```diff
@@ -37,34 +57,45 @@ gen-mocks: check-mockery
 	mockery -output pkg/fileutils/mocks					-outpkg mocks				-dir pkg/fileutils																				-name Copier
 
 install: build
-	cp $(OUTPUT) $(GOBIN)
+	$(COPY_CMD) $(OUTPUT)$(OUTPUT_EXT) $(GOBIN)
 
-$(OUTPUT): $(GO_SOURCES) vendor VERSION
-	go build -o $(OUTPUT) -ldflags "$(LDFLAGS_VERSION)"
+$(OUTPUT): vendor VERSION
+	go build -o $(OUTPUT)$(OUTPUT_EXT) -ldflags "$(LDFLAGS_VERSION)"
 
-release: $(GO_SOURCES) vendor VERSION
+release: vendor VERSION
+ifeq ($(ISWIN),true)
+	cmd /C "set GOOS=windows&&set GOARCH=amd64&& go build -ldflags "$(LDFLAGS_VERSION)" -o $(OUTPUT)$(OUTPUT_EXT) && 7z a -tzip riff-windows-amd64.zip $(OUTPUT)$(OUTPUT_EXT) && del /q $(OUTPUT)$(OUTPUT_EXT)"
+	cmd /C "set GOOS=darwin&&set GOARCH=amd64&& go build -ldflags "$(LDFLAGS_VERSION)" -o $(OUTPUT) && 7z a -ttar riff-darwin-amd64.tar $(OUTPUT) && 7z a -tzip riff-darwin-amd64.tgz riff-darwin-amd64.tar && del /q riff-darwin-amd64.tar && del /q $(OUTPUT)"
+	cmd /C "set GOOS=linux&&set GOARCH=amd64&& go build -ldflags "$(LDFLAGS_VERSION)" -o $(OUTPUT) && 7z a -ttar riff-linux-amd64.tar $(OUTPUT) && 7z a -tzip riff-linux-amd64.tgz riff-linux-amd64.tar && del /q riff-linux-amd64.tar && del /q $(OUTPUT)"
+else	
 	GOOS=darwin   GOARCH=amd64 go build -ldflags "$(LDFLAGS_VERSION)" -o $(OUTPUT)     && tar -czf riff-darwin-amd64.tgz $(OUTPUT) && rm -f $(OUTPUT)
 	GOOS=linux    GOARCH=amd64 go build -ldflags "$(LDFLAGS_VERSION)" -o $(OUTPUT)     && tar -czf riff-linux-amd64.tgz $(OUTPUT) && rm -f $(OUTPUT)
-	GOOS=windows  GOARCH=amd64 go build -ldflags "$(LDFLAGS_VERSION)" -o $(OUTPUT).exe && zip -mq riff-windows-amd64.zip $(OUTPUT).exe && rm -f $(OUTPUT).exe
+	GOOS=windows  GOARCH=amd64 go build -ldflags "$(LDFLAGS_VERSION)" -o $(OUTPUT)$(OUTPUT_EXT) && zip -mq riff-windows-amd64.zip $(OUTPUT)$(OUTPUT_EXT) && rm -f $(OUTPUT)$(OUTPUT_EXT)
+endif
 
 docs: $(OUTPUT) clean-docs
```

Comment:
This is why it's better to merge OUTPUT and OUTPUT_EXT: this is a *dependency*, needs to look at the actual produced file mod date